integration/stability/stability_test.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This is the stability test for the new `agent`, right? What about if we call this script: agent_stability_test.sh? or something similar? We may have more stability tests in the future for other components.